docs/recipes/recipe-marketing-attribution/index.md

```diff
@@ -5,113 +5,79 @@ date: "2020-10-12"
 
 ## Introduction
 
-Attribution modeling is the process of assigning credit for conversions to marketing touch points. The key to attribution modeling is capturing all marketing touchpoints and all conversions, and being able to assign them to a specific user. This allows you to look at the effectiveness of your marketing spend across platforms and channels, over time.
-
-This recipe allows you to explore how you can get started with taking control of your attribution modeling with Snowplow. Owning your attribution modeling forces you make assumptions explicitly and deliberately, a crucial step in understanding its limitations and using its outputs appropriately. Furthermore, being able to run multiple attribution models in parallel allows you to see the impact different modeling logic have on the outputs of the model.
-
-## What you'll be doing
-
-You have already set up Snowplow’s out of the box web tracking by instrumenting the JavaScript Tracker in your application. This includes tracking `page_view` and `page_ping` events.
+[Attribution modeling](https://snowplow.io/use-cases/marketing-attribution/) is the process of assigning credit for conversions to marketing touch points, vital for every company's acquisition strategy. This is the first use case many companies deploy with Snowplow for a few reasons:
+* Snowplow delivers very high quality event data and concurrent multitouch attribution dbt models out of the box
+* No custom tracking is required for a first iteration
+* Configurability is the key to attribution, with Snowplow everything from the conversion trigger to the session definition and touchpoint weighting algorithm is configurable
 
-With each page view, Snowplow automatically captures the marketing parameters and referrer information as well as a session identifier. This means you have already started capturing marketing touches.
+By the end of this guide you will have built a table just like this one for your business:
+![](images/outcome.png)
 
-To attribute conversions, you’ll also need to track conversion events. You'll then be able to run simple query that attributes the conversions to the marketing channels based on three popular marketing attribution models:
 
-- First touch: this model gives all credit to the user's first touch preceding a conversion
-- Last touch: this model gives all credit to the user’s last touch preceding a conversion
-- Linear: this model gives equal credit to all of the user touches preceding a conversion
 
-## Design and implement the `conversion` event
+## What you'll be doing
+The most important goal of this guide is to help you learn something new about your customers!
 
-#### Designing the `conversion` event
+:::note
+Prerequisites: None!
 
-We have already created a custom `conversion` event for you in [Iglu Central](http://iglucentral.com/).
+You don't need any tooling to build a v1 model, you will need a warehouse and dbt to do the more advanced final section of this guide.
+:::
 
```

Comment:
This feels a little disingenuous as the requirements increase throughout the recipe, for step 3 you need to be a BDP Cloud customer. Could we call out the change between step 2 and step 3, or be really clear that you can see the output and everything by just getting to step 2 which can all be done with Try Snowplow. My worry is if I skim a tutorial and see I need a credit card for a step, I might not try the tutorial at all.

Reply:
Thats fair, I have tried to distinguish the two now